components/Map/MapMarker.tsx

```diff
@@ -0,0 +1,31 @@
+import { Text } from '@chakra-ui/react'
+import { InfoWindow, Marker, MarkerProps } from '@react-google-maps/api'
+import { useState } from 'react'
+
+export const MapMarker = ({position, title}: MarkerProps): JSX.Element => {
+  const marker = new google.maps.Marker()
+  marker.setOptions({
+    anchorPoint: new google.maps.Point(0, -40),
+  })
```

Comment:
Needed to offset the `InfoWindow` component from the marker.
- [@react-google-maps/api docs](https://react-google-maps-api-docs.netlify.app/#infowindow)
- [Google Maps API docs](https://developers.google.com/maps/documentation/javascript/reference/info-window#InfoWindow.open)